packages/manager/src/services/ephemeral/queries.tsx

```diff
@@ -0,0 +1,26 @@
+import { orchestratorReq } from '@app/config/orchestratorReq';
+import { useQuery } from '@tanstack/react-query';
+import { TEphemeralEnv } from './types';
+
+const ephemeralQueryKeys = {
+  list: (webPropertyName: string) => ['ephemeral-env', webPropertyName] as const
+};
+
+const path = `/webproperty/eph/list?propertyName=`;
+
+const fetchEphemeralEnvironments = async (propertyName: string): Promise<TEphemeralEnv[]> => {
+  const { data } = await orchestratorReq.get(`${path}${propertyName}`);
```

Comment:
There is a query option in .get function. Use that please and u can place the path directly here as its clear and not used anywhere